api/v1beta1/verticadb_types.go

```diff
@@ -499,9 +499,29 @@ type Subcluster struct {
 	// at least one primary subcluster in the database.
 	IsPrimary bool `json:"isPrimary"`
 
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:hidden"
+	// Internal state that indicates whether this is a standby subcluster for a
+	// primary.  Standby are transient subclusters that are created during an
+	// online image change.
+	IsStandby bool `json:"isStandby,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:hidden"
+	// If a standby, this is the name of the primary subcluster it is a standby
```

Comment:
Isn't there a missing word on this line?

Reply:
How about this?

```
// If this is a standby, this is the name of the primary subcluster the
// subcluster was created for.  This is state internally managed for
// an online image change. 
```

Reply:
Yeah I think it is better. Maybe because my english is no so good but I feel like having 2 consecutive occurrences of subscluster is repetitive. What do you thin about:
```

// If this is a standby subcluster, this is the name of the primary subcluster 
// it was created for.  This is state internally managed for
// an online image change. 
```